content/en/security_platform/application_security/_index.md

```diff
@@ -1,17 +1,47 @@
 ---
-title: Monitoring Threats with Datadog Application Security
+title: Application Security
 kind: documentation
+is_beta: true
+disable_sidebar: true
 further_reading:
 - link: "/security_platform/guide/how-appsec-works/"
   tag: "Guide"
-  text: "How Datadog Application Security Monitoring Works"
+  text: "Learn how Application Security works"
 - link: "/security_platform/application_security/setup_and_configure/#compatibility"
   tag: "Documentation"
-  text: "Programming Language and Framework Compatibility"
+  text: "Learn more about language and framework compatibility"
 ---
 
+<div class="alert alert-warning">
+Application Security is currently in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
+</div>
 
+{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="The Application Security landing page in Datadog, which displays your services, suspicious requests, and detected signals" width="75%">}}
 
-## Further Reading
+Datadog Application Security provides protection against application-level threats by identifying and blocking attacks that target code-level vulnerabilities, such as SQL injections and cross-site scripting (XSS) exploits.
 
-{{< partial name="whats-next/whats-next.html" >}}
\ No newline at end of file
+Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever a vulnerability or attack occurs.
+
+For example, if a threat is detected, a security signal is generated in Datadog, which triggers notifications in-app and for your configured apps or services. Your team can access these signals in Datadog and begin investigation by leveraging the deep observability data provided by Application Security and APM distributed tracing in one panel.
+
+## Configure your environment
+
+Powered by provided [out-of-the-box rules][3], Application Security detects threats without the manual configuration of rules for an Agent or provider. If you're already using Datadog [APM][1], setup only requires implementation of one environment variable to get started.
```

Comment:
* detection rules
*of rules for an Agent or provider: not clear nor useful to me.

> If you're already using Datadog [APM][1], setup only requires implementation of one environment variable to get started.
Env variable are set, not technically implemented